### ISSUE TITLE
Alter the README.md to fulfill the assignment guidelines

### ISSUE THREAD

**Issue by tis22** [heyitsalina/ant_search_algorithm#79]
According to the project description we should provide an explaination (for developers!) about the code-structure and how the software can be used.

**Comment by tis22** [comment]
You may use shields.io for a nice outcome! 👀

**Comment by honnigmelone** [comment]
I will look into it :)

**Comment by tis22** [comment]
I saw, that you already closed this issue!

The developer documentation should (as we were told and as I understand it) go into a little more detail about how the code is structured and works (i.e. a brief outline of how the main functions and files build on each other and work together). 

That's why I reopened the issue. 😓 

The current status is more suitable for #80 but the functions/settings that the user can use should also be explained more. (e.g. how exactly he can change the colony size or food size, not just that he can -> that he has to click on the pictures etc.)
